### DISCUSSION TITLE
Pixel Glow Doesn't Go Well with Masque

### DISCUSSION THREAD

**Discussion by Causese** [SFX-WoW/Masque#196] [Issues] [answered]
hello there,

weakauras introduced a new pixel glow feature that uses
https://www.curseforge.com/wow/addons/libcustomglow

enabling glow in weakauras, without masque, looks like this:
https://i.gyazo.com/0e00357a83907a9f2698afd18ef5275d.gif

with masque (flatsquares)
https://i.gyazo.com/0b96d62b273cbec65a62fe86e81b1485.gif

i was hoping that you could find a way to make the masque glow appear at a similar location

**Comment by StormFX** [comment, marked as answer]
_Masque_ doesn't support custom regions that third-party add-ons create. This would be an issue on WeakAuras end.
